html/pfappserver/lib/pfappserver/I18N/i_default.po

```diff
@@ -1 +0,0 @@
-en.po
```

Comment:
Not sure that you should modify this to put the strings back in it. My understanding was that en.po was the only place where they needed to be